tests/unit/helpers/eventsRepetition.test.js

```diff
@@ -315,6 +323,33 @@ describe('createRepeatedEvents', () => {
       ]
     );
   });
+
+  [INTERNAL_HOUR, UNAVAILABILITY].forEach((type) => {
+    it(`should not call customerFindOne if eventType is ${type} and not intervention`, async () => {
```

Comment:
Tu peux mettre `should not check if customer is stopped if event is ${type} and not intervention` qu'en penses-tu ?